compaction/time_window_compaction_strategy.cc

```diff
@@ -106,7 +106,23 @@ time_window_compaction_strategy_options::time_window_compaction_strategy_options
 time_window_compaction_strategy_options::time_window_compaction_strategy_options(const time_window_compaction_strategy_options&) = default;
 
 void time_window_compaction_strategy_options::validate(const std::map<sstring, sstring>& options, std::map<sstring, sstring>& unchecked_options) {
+    validate_compaction_window_unit(options);
+    unchecked_options.erase(COMPACTION_WINDOW_UNIT_KEY);
 
+    validate_compaction_window_size(options);
+    unchecked_options.erase(COMPACTION_WINDOW_SIZE_KEY);
+
+    validate_expired_sstable_check_frequency_seconds(options);
+    unchecked_options.erase(EXPIRED_SSTABLE_CHECK_FREQUENCY_SECONDS_KEY);
+
+    validate_timestamp_resolution(options);
+    unchecked_options.erase(TIMESTAMP_RESOLUTION_KEY);
+
+    unchecked_options.erase("enable_optimized_twcs_queries");
+    unchecked_options.erase("min_threshold");
+    unchecked_options.erase("max_threshold");
+    // TODO: Do we want to allow unsafe_aggressive_sstable_expiration?
+    unchecked_options.erase("unsafe_aggressive_sstable_expiration");
```

Comment:
Not listed in docs nor parsed.
Compare: https://github.com/scylladb/scylla-tools-java/blob/eb3c43f8e069334f24d8aa9efd3cb350fe015aa7/src/java/org/apache/cassandra/db/compaction/TimeWindowCompactionStrategyOptions.java#L164

Reply:
@raphaelsc please advise

Reply:
We can allow it for compatibility. It's fine to leave it as is, as we're recognizing its existence, despite we're not using it.